examples/pytorch/bert/pytorch_job_bert.yaml

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Is it fine that we have separate Licence here ?

Reply:
Same concern here. Not sure why a simple bert example has companies license here